PhysicsTools/NanoAOD/plugins/GenWeightsTableProducer.cc

```diff
@@ -136,6 +136,8 @@ namespace {
     void clear() {
       for (auto x : countermap)
         x.second.clear();
+      active_el = nullptr;
```

Comment:
please run `scram b code-format` after all modifications to adapt the code structure to the requirements of the upstream repository

Reply:
it doesn't seem to work:

scram b code-format
Reading cached build data
>> Local Products Rules ..... started
>> Local Products Rules ..... done
Unknow target code-format